components/influxdb/scripts/upgrade_validation.py

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env python
+
+from os.path import join, dirname
+from cloudify import ctx
+
+ctx.download_resource(
+    join('components', 'utils.py'),
+    join(dirname(__file__), 'utils.py'))
+
+import utils  # NOQA
+
+INFLUX_SERVICE_NAME = 'influxdb'
+
+utils.upgrade_validation_directories(INFLUX_SERVICE_NAME)
+
+if not utils.systemd.is_alive(INFLUX_SERVICE_NAME):
+    ctx.abort_operation('InfluxDB service must be running to allow data '
+                        'migration')
```

Comment:
This should change and be done for all services (except for the webui maybe). This is because pretty soon the upgrade process would run run a system workflow to create a snapshot and this would require all components to be running.